machine1/dbus.go

```diff
@@ -110,6 +112,16 @@ func (c *Conn) getPath(method string, args ...interface{}) (dbus.ObjectPath, err
 	return path, nil
 }
 
+// CreateMachine creates the container with the systemd-machined
```

Comment:
`CreateMachine creates a new virtual machine or container with systemd-machined, generating a scope unit for it`.